core/src/test/java/org/elasticsearch/index/percolator/ExtractQueryTermsServiceTests.java

```diff
@@ -218,6 +220,31 @@ public void testExtractQueryMetadata_boostQuery() {
         assertThat(terms.get(0).bytes(), equalTo(termQuery1.getTerm().bytes()));
     }
 
+    public void testExtractQueryMetadata_commonTermsQuery() {
```

Comment:
So long as you go through the builders I don't care how you do it. I'm happy with something in PercolatorQueryTests.